docs/framework/unmanaged-api/debugging/closeclrenumeration-function-for-silverlight.md

```diff
@@ -0,0 +1,65 @@
+---
+description: "Learn more about: CloseCLREnumeration Function for Silverlight"
+title: "CloseCLREnumeration Function for Silverlight"
+ms.date: "03/30/2017"
+api_name: 
+  - "CloseCLREnumeration"
+api_location: 
+  - "dbgshim.dll"
+api_type: 
+  - "COM"
+f1_keywords: 
+  - "CloseCLREnumeration"
+helpviewer_keywords: 
+  - "debugging API [Silverlight]"
+  - "Silverlight, debugging"
+  - "CloseCLR Enumeration function"
+ms.assetid: 5e3c3958-80bb-43b1-a96b-dd3e6dbd9cd7
+topic_type: 
+  - "apiref"
+---
+# CloseCLREnumeration Function for Silverlight
```

Comment:
I would love to delete all the Silverlight APIs here,  but I'm not sure the docs policy on out of support APIs.

Reply:
The rule is that all APIs called in Microsoft high-volume products (HVPs) must be publicly documented. So it may not be necessary to document the Silverlight APIs, but it would be hard to know.

Also, a single doc for multiple versions is fine. You could even link to that doc from this section of the TOC if that is the look you're going for.

Reply:
I'm going to leave the silverlight docs alone.